gadgets/fsnotify/program.bpf.c

```diff
@@ -303,6 +331,36 @@ int BPF_KRETPROBE(inotify_handle_inode_event_x, int ret)
 	return 0;
 }
 
+// Linux < 5.11 does not have inotify_handle_inode_event but inotify_handle_event
+// https://github.com/torvalds/linux/commit/1a2620a99803ad660edc5d22fd9c66cce91ceb1c
+SEC("kprobe/inotify_handle_event")
+int BPF_KPROBE(inotify_handle_event_e, struct fsnotify_group *group,
+	       struct inode *inode)
+{
+	if (!fanotify_only) {
+		u64 pid_tgid = bpf_get_current_pid_tgid();
+		struct fsnotify_insert_event_value value = {
+			.type = inotify,
+			.i_ino = 0, // FIXME: not available on Linux < 5.11
```

Comment:
If this needs a fix, could you open an Issue?

Reply:
Done: https://github.com/inspektor-gadget/inspektor-gadget/issues/4222